tests/sclT0009-unused-symbols/sym01.exec

```diff
@@ -0,0 +1,2 @@
+# format: exec
+exec:
```

Comment:
This looks like a great test :+1: 
But it doesn't check whether the output matches the expected value?